tests/test_cnvkit.yml

```diff
@@ -0,0 +1,158 @@
+- name: Run variant calling on somatic samples with cnvkit
```

Comment:
Might consider splitting these into their own individual files `tests/workflows/cnvkit/test_variantcalling_somatic.yml` or something for readability, these are going to get quite long.

Reply:
I don't think there will be much more tests for cnvkit actually, but I'll think about that.
Thanks